src/components/common/input.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+const Input = props => {
```

Comment:
Always skip line between imports and components

Reply:
I have added the line. The reason I didn't observe this was, because of my linter for complexity, so visually you could see space between when it's not there. I have disabled it